packages/non-core/facebook/deprecation_notice.js

```diff
@@ -0,0 +1,9 @@
+console.warn(
```

Comment:
Yeah, it is and that's the right place for them.  I put this here until we could discuss it and move them all at once.